spotify/models/common.py

```diff
@@ -107,3 +113,100 @@ def __repr__(self):
 
     def __str__(self):
         return self.id
+
+
+class TokenInfo:
```

Comment:
Maybe a better name would be `CachedToken`, TokenInfo reads a little ambiguously to me.

Reply:
Hmm, I'm not sure about this. `CachedToken` would imply that the token always will be associated with a cache file which in most cases will not be the case. If `TokenInfo` is too ambiguous, maybe something like `SpotifyToken` or `SpotifyAuthentication`?